centreon/src/Utility/Interfaces/UUIDGeneratorInterface.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Utility\Interfaces;
+
+interface UUIDGeneratorInterface
```

Comment:
Even for acronyms, we don't need CamelCase ?
```suggestion
interface UuidGeneratorInterface
```

Reply:
Why not calling this interface UidGeneratorInterface ?
If we need other "uid" like ULID, GUID, ...

Reply:
as its a acronym i think Uppercase is better (to follow what we do for API instead of Api for example)